sidekiq_fast_debounce.gemspec

```diff
@@ -39,7 +39,7 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'sidekiq'
 
-  spec.add_development_dependency 'bundler', '~> 1.17'
+  spec.add_development_dependency 'bundler'
```

Comment:
do you need a _minimum_ version of bundler?

Reply:
Don't think so. It's a dev dependency, doubtful that whatever someone is using is going to not work.